.pre-commit-config.yaml

```diff
@@ -28,10 +28,12 @@ repos:
     - id: shfmt
       args: ["-w", "-i", "4", "-ci"]
 
-- repo: https://github.com/google/yapf
```

Comment:
Let's also drop `.style.yapf` now.

Overall a surprisingly small diff over yapf, I am glad we didn't customize it too much back then.